utils/generateJwtToken.js

```diff
@@ -1,23 +1,19 @@
 require("dotenv").config();
 const jwt = require("jsonwebtoken");
 
-const TOKEN_EXPIRATION_DEFAULT = "5d";
+const TOKEN_EXPIRATION_DEFAULT = "1h";
```

Comment:
Would it be better if we set this `TOKEN_EXPIRATION_DEFAULT` in the `.env` file, so we'd have a single goto place for all things related to token expirations?

Reply:
I prefer to keep values that change based on environments in the `.env` file. This value won't change across environments so I think we should leave it in code.

I can move it the `constants.js` file in a subsequent PR.

Reply:
I agree with this logic.